internal/backend/s3/config.go

```diff
@@ -22,6 +22,7 @@ type Config struct {
 
 	Connections uint `option:"connections" help:"set a limit for the number of concurrent connections (default: 5)"`
 	MaxRetries  uint `option:"retries" help:"set the number of retries attempted"`
+	Region	     string
```

Comment:
You have a TAB character in there, causing the `go fmt` test to fail. Please run your file through `go fmt`.